files/justfiles/secureblue.just

```diff
@@ -682,3 +682,329 @@ debug-info:
+                    resolver_hostname="p0.freedns.controld.com"
+                    resolver_https_address="https://freedns.controld.com/p0"
+                    ;;
+                1)
+                    resolver_ipv4_address="76.76.2.1"
+                    resolver_ipv4_address_2="76.76.10.1"
+                    resolver_ipv6_address="2606:1a40::1"
+                    resolver_ipv6_address_2="2606:1a40:1::1"
+                    resolver_hostname="p1.freedns.controld.com"
+                    resolver_https_address="https://freedns.controld.com/p1"
+                    ;;
+                2)
+                    resolver_ipv4_address="76.76.2.2"
+                    resolver_ipv4_address_2="76.76.10.2"
+                    resolver_ipv6_address="2606:1a40::2"
+                    resolver_ipv6_address_2="2606:1a40:1::2"
+                    resolver_hostname="p2.freedns.controld.com"
+                    resolver_https_address="https://freedns.controld.com/p2"
+                    ;;
+                3)
+                    resolver_ipv4_address="76.76.2.3"
+                    resolver_ipv4_address_2="76.76.10.3"
+                    resolver_ipv6_address="2606:1a40::3"
+                    resolver_ipv6_address_2="2606:1a40:1::3"
+                    resolver_hostname="p3.freedns.controld.com"
+                    resolver_https_address="https://freedns.controld.com/p3"
+                    ;;
+                4)
+                    resolver_ipv4_address="76.76.2.4"
+                    resolver_ipv4_address_2="76.76.10.4"
+                    resolver_ipv6_address="2606:1a40::4"
+                    resolver_ipv6_address_2="2606:1a40:1::4"
+                    resolver_hostname="family.freedns.controld.com"
+                    resolver_https_address="https://freedns.controld.com/family"
+                    ;;
+            esac
+            ;;
+        2)
+            resolver_has_second_ip="n"
+            resolver_supports_ipv6="y"
+            echo "Setting resolver Mullvad."
+            echo "What content would you like to filter:"
+            echo "    0) No filtering"
+            echo "    1) Ad and tracker filtering"
+            echo "    2) Previous plus malware filtering"
+            echo "    3) Previous plus social media filtering"
+            echo "    4) Previous plus gambling adult content filtering, without social media"
+            echo "    5) Previous plus social media filtering"
+            while [[ "$valid_input" == "0" ]]; do
+                read -p "Selection [0-5]: " resolver_subselection
+                if [[ "$resolver_subselection" == [012345]* ]]; then
+                    valid_input="1"
+                else
+                    echo "That is not a valid selection."
+                fi
+            done
+            valid_input="0"
+            case "$resolver_subselection" in
+                0)
+                    resolver_ipv4_address="194.242.2.2"
```

Comment:
for all providers, is there any way to get these programmatically? since it's possible they change in the future?